templates/registration/login.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <form method="post" action="{% url 'login' %}" style="margin: 35px" novalidate>
```

Comment:
The form action is set to `{% url 'login' %}`, which is correct for directing the form submission to the login URL. Ensure that the URL name 'login' is correctly defined in your URL configurations.